src/data/matrix/basic.lean

```diff
@@ -397,6 +445,13 @@ def ring_hom.map_matrix [decidable_eq m] [semiring α] {β : Type w} [semiring 
 
 open_locale matrix
 
+/-- Specialize `ring_hom.map_mul` to use `matrix.mul` instead of `has_mul.mul`. -/
+@[simp]
+lemma map_matrix.map_mul [decidable_eq m] [semiring α] {β : Type w} [semiring β]
```

Comment:
Should this be
```suggestion
lemma ring_hom.map_matrix.map_mul [decidable_eq m] [semiring α] {β : Type w} [semiring β]
```
so that its name is consistent with `ring_hom.map_matrix`?